src/timeline/js/functions.js

```diff
@@ -58,7 +58,7 @@ function drawAudio(scope, clip_id) {
   //get the clip in the scope
   var clip = findElement(scope.project.clips, "id", clip_id);
 
-  if (clip.show_audio) {
+  if (clip.ui && clip.ui.audio_data) {
```

Comment:
I think this function `drawAudio` should be converted to a Angular `watch`, in the `clip` directive. 

Watch:
- scale changes
- clip UI : audio_data changes